components/api/upgrade-org.ts

```diff
@@ -14,3 +14,7 @@ import { mapleClient } from "./maple-client"
 export async function upgradeOrganization(userId: string) {
   return mapleClient.patch(`/api/users/${userId}`, { role: "organization" })
 }
+
+export async function downgradeOrganization(userId: string) {
```

Comment:
I suppose the user doesn't _technically_ have to be an organization for this function to work... maybe just `setAsUserRole`?